README.adoc

```diff
@@ -1028,37 +1028,30 @@ end
 
 === Context Descriptions
 
-When an example block description is composed with `context` block descriptions, it should result in a sentence with proper grammar.
-This typically means the `context` should start with a term such as 'when', 'with', or 'without'.
+Context descriptions should describe the conditions shared by all the examples within. Full example names—formed by concatenation of all nested block descriptions—should form a readable sentence.
```

Comment:
Not sure, maybe typographically dash is suitable here, I'm not sure how any of the guides use something except ASCII. Would it be ok to put this clarification in parentheses?

In any case it's a bit to dense when rendered:
![image](https://user-images.githubusercontent.com/6916/72941296-a7972f00-3d81-11ea-9680-34aa541d7c6e.png)

Reply:
I think the dash is most correct, but parentheses are also acceptable. Changed to parentheses.